pyramid_debugtoolbar/static/js/toolbar.js

```diff
@@ -59,14 +60,22 @@ $('#settings .switchable').click(function() {
   }
 });
 
-// $(".pDebugSortable").tablesorter();
 
-bootstrap_panels = ['pDebugVersionPanel', 'pDebugHeaderPanel']
+try {
```

Comment:
why would this fail? Please at least add a comment here about where it's expected to fail.